pages/partials/submission.ejs

```diff
@@ -43,7 +43,7 @@
     </button>
   </div>
 
-  <div class="collapse js-submission-body <% if (submission.submission_number == submissionCount || locals.expanded) { %>show<% } %> <% if (submissionHtml === undefined) { %>render-pending<% } %>" data-submission-id="<%= submission.id %>" id="submission-<%= submission.id %>-body">
+  <div class="collapse js-submission-body <% if (submission.submission_number == submissionCount || locals.expanded) { %>show<% } %> <% if (submissionHtml === undefined && question.type === 'Freeform') { %>render-pending<% } %>" data-submission-id="<%= submission.id %>" id="submission-<%= submission.id %>-body">
```

Comment:
I happened to discover this while testing - we've been unable to display older submissions for v2 questions for a while now! I suppose it's a promising sign that no one noticed 😂

Reply:
Ha, nice! 😄